bevy_lint/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Install `bevy_lint`
+author: BD103
+description: |
+  Installs the Bevy linter and its required toolchain.
+
+  This action currently uses Rustup to install the required toolchain components, and it uses
+  `cargo install` to install the linter itself. The linter will be installed to `~/.cargo/bin` and
+  will be available through the `PATH` by default.
+
+  Note that this currently will override the default Rustup toolchain to nightly. You can fix this
+  by calling `rustup default` after this action is run.
```

Comment:
Should we integrate `rustup default` directly in this script? Or are there scenarios where you don't want to use it?

Reply:
I tried to make the action restore the original default toolchain, but my first attempt didn't work. I'm going to leave this to a follow-up issue.

Reply:
Have you considered not using the action, instead requiring that `rustup` and `cargo` are already installed and then manually installing the required toolchain via the `rustup` command?

But that can indeed be done in a follow-up :)

Reply:
Interesting! That does sound a lot nicer, although it would mean ditching [`dtolnay/rust-toolchain`](https://github.com/dtolnay/rust-toolchain) since it's just not configurable enough to install Rustup without a specific toolchain. I think it's worth considering, but definitely later